wordpress-mobile/gutenberg-mobile
# Uses the global store for the post title and adds undo/redo support.

## Description:

Following @Tug 's indications [here](
https://github.com/wordpress-mobile/gutenberg-mobile/pull/450#discussion_r248672329), this PR wires our title to the global store title.

This PR also adds undo/redo support for the title.

Moves forward #372 .

## Dependencies:

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/13514

## Testing:

### Testing edits:

1. Launch the demo app.
2. Make sure the initial title is properly set up.
3. Edit the title, switch to HTML mode and back.

Make sure the title is still as you left it before switching modes.

### Testing undo redo:

1. Edit the title as you see fit.
2. Use the undo / redo buttons from the toolbar.

Make sure undo / redo works as it does in web-Gutenberg (ie: that's our baseline for comparing).
